neo/io/blackrockio_v4.py

```diff
@@ -386,7 +384,7 @@ def __extract_nsx_file_spec(self, nsx_nb):
             spec = '{}.{}'.format(
                 nsx_file_id['ver_major'], nsx_file_id['ver_minor'])
         else:
-            raise IOError('Unsupported NSX file type.')
+            raise OSError('Unsupported NSX file type.')
```

Comment:
This should not be an `OSError`. I suggest leaving it as `IOError` for now, then we can replace it with something suitable, e.g. a custom `NotSupportedError`, in future.